invokeai/backend/model_manager/load/load_base.py

```diff
@@ -21,7 +24,9 @@
 
 @dataclass
 class LoadedModel:
-    """Context manager object that mediates transfer from RAM<->VRAM."""
+    """
+    Context manager object that mediates transfer from RAM<->VRAM.
+    """
```

Comment:
This class is pretty unusual, so I'm going to ask for lots of documentation.

Things I think we should add to this docstring:
- Explain why there are two different context managers in one here.
- Mention that the 'old' context manager is deprecated. I think we want to move toward only supporting `model_on_device()`.

Reply:
Yeah, it’s two context managers in one! Shall I introduce a deprecation warning? 

There are a lot of places where the old context manager are used. How urgent to upgrade it everywhere?  This is something I’d rather do in a subsequent PR.

Reply:
Note is applied.